script/lib/util.py

```diff
@@ -30,8 +30,7 @@
   os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
 )
 SRC_DIR = os.path.abspath(os.path.join(__file__, '..', '..', '..', '..'))
-BOTO_DIR = os.path.abspath(os.path.join(__file__, '..', '..', '..', 'vendor',
-                                        'boto'))
+BOTO_DIR = os.path.join(ELECTRON_DIR, 'vendor', 'boto')
```

Comment:
Is this needed anymore?

Reply:
it was, until i removed the last usage of boto in the next commit